packages/rpc-core/src/types/rpc-methods/common.d.ts

```diff
@@ -0,0 +1,11 @@
+declare type DataSlice = readonly {
+    offset: number;
+    length: number;
+};
+
+// TODO: Eventually move this into whatever package implements transactions
+declare type Finality = 'confirmed' | 'finalized' | 'processed';
```

Comment:
nit: Is `Finality` the new term? If so, we should probably update the docs and all that. I'd prefer to keep this `Commitment` to make mental mapping easier, but I won't put up a big fight about it.

Reply:
Ooh. I thought these three were what we're calling finality, as opposed to the olde deprecated suite of commitments.

From web3.js today:

https://github.com/solana-labs/solana-web3.js/blob/ad23683e8a42c726995cf0c1f0f903b20152854f/packages/library-legacy/src/connection.ts#L476-L501

So yeah, I have this wrong, somehow. I'll rename it.